news/2017-10-28-beatmap-spotlights-september-2017.md

```diff
@@ -0,0 +1,114 @@
+
+
+## osu!
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/reloaded.jpg)](https://osu.ppy.sh/s/380143)
+
+[Handsome](https://osu.ppy.sh/u/2123087)'s latest ranked map, RELOADED, works most effectively thanks to its limited framework. The map is designed around a strict symmetrical object placement gimmick, along with the restriction of almost all sliders moving only upwards. Through this design, it provides an engaging experience that no other maps can provide!
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/we-are-one.jpg)](https://osu.ppy.sh/s/605745)
+
+In his latest mapset, [Frey](https://osu.ppy.sh/u/695678) presents us a fast-paced dubstep piece by renowned artist Pegboard Nerds. Five full difficulties of non-stop intense gameplay, all mapped fully by none other than himself. The very nature of the set brings us a genuine feeling of coherence and unity, a rare sight in the modern jungle of collaborations and guest difficulties.
+
+From the ‘Normal’ difficulty we are already introduced to his characteristic clean style, which relies heavily on visual appeal by minimizing overlaps and prioritizing blankets. The style becomes more pronounced as we progress up the difficulties, with denser rhythms and spacing freedom allowing for a more distinct aesthetic. Everything culminates in the 'Night Begins' difficulty, where we see energetic patterns left and right, all arranged in an elegant, simple fashion.
+
+A charming set that shows that one does not have to break boundaries to secure a spot in the spotlights, as even the simplest of styles, when mastered, are able to do so!
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/beginnings.jpg)](https://osu.ppy.sh/s/646603)
+
+Inspired to come out of mapping retirement, [La Cataline](https://osu.ppy.sh/u/301279)’s guest difficulty adds additional charm to [Mirash](https://osu.ppy.sh/u/2841009)'s tantalising mapset of her song - “Beginnings”. This flagship difficulty has a taste from times long since past, but easily holds up to the modern standards of mapping. 
+ 
+Don’t discount the rest of the spread though, especially not the ‘Collab Insane’, which consists of varied unique patterns and interesting stylistic choices! With such diverse talent, this is one gem you certainly don’t want to miss!
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/chrono-diver.jpg)](https://osu.ppy.sh/s/419600)
+
+**[toybot](https://osu.ppy.sh/u/2848604)**, [ProfessionalBox](https://osu.ppy.sh/u/3280792), [Sing](https://osu.ppy.sh/u/3795679) & [byfar](https://osu.ppy.sh/u/4674054). A quick glance at the collaborator's names would already tell you that you're in for an extravagant composition. Filled with modern patterns and energy-filled gameplay, this mapset is accompanied by an alluring storyboard put together by famous storyboarders [-Tochi](https://osu.ppy.sh/u/3664366) and [Storyboarder](https://osu.ppy.sh/u/6124671)!
```

Comment:
The image should probably include byfar, probox and toybot too

Reply:
Probably, but due to ProfessionalBox's long name its impossible to fit in. And I won't short his username either.